### DISCUSSION TITLE
Sonic Pad or Pi alternatives?

### DISCUSSION THREAD

**Discussion by timarland** [bassamanator/Sovol-SV06-firmware#7] [Q&A]
Will this work with the Creality Sonic Pad (klipper pad) or something like an Orange Pi since rPI are so hard to get now?  Thanks!

**Comment by bassamanator** [comment]
Works on any machine running linux, with klipper installed of course. Someone was running this on the Sonic pad, but they did report some issue that I can't specifically recall. 

Report any issues if you encounter any with a sonic pad.

Also just created a discussion topic on this matter [here](https://github.com/bassamanator/Sovol-SV06-firmware/discussions/8), please check.

**Comment by davetyra** [comment]
I am currently running your printer.cfg and other .cfgs on my Sonic Pad, I had been running a more basic printer.cfg at first and changed to yours for the better macro support. Thanks for all the effort! I had to make some changes for the Sonic Pad, I can write up what I did if anyone is interested.

**Reply by bassamanator** [reply]
Yes, please let us know what changes were required, so that others might have an easier experience. I'm trying to provide a Klipper experience to 3D printing beginners, and it's not easy.
